### ISSUE TITLE
criterion JSON-LD Questions

### ISSUE THREAD

**Issue by cbizon** [clingen-data-model/clingen-interpretation#72]
```
"criterion": {
            "cg:id": "PM3",
            "cg:type": "Criterion",
            "description": "For recessive disorders, detected in trans with a pathogenic variant",
            "shortDescription": "Recessive in trans with pathogenic variant",
            "defaultStrength": "CG-evidence-strength:pm",
            "targetInterpretation": "LN:LA6668-3"
          },
```

1. what is the targetInterpretation for / what does it mean?
2. I assume that PM3 is going to be replaced with an IRI?
3. Is this all going to be a codable concept?

**Comment by rrfreimuth** [comment]
FWIW, the code in targetInterpretation is a LOINC code for “pathogenic”.  See https://r.details.loinc.org/AnswerList/LL4034-6.html

Re: #3, there are two pieces of information here:  the criterion and the interpretation linked to that criterion.  I suppose it would be possible to create a single codeable concept that contained both pieces, but wouldn’t they be better represented as separate entities (each a codeable concept)?  Sorry if I am misunderstanding the question.

Thanks,
Bob

**Comment by cbizon** [comment]
I think you're right.  In the spreadsheet we have them as different attributes (strength and target).  But in the JSONLD we have target, but then strength looks like a combination of the two?  Keeping them separate makes the most sense to me - perhaps this is just a work in progress.  @bpow what do you think?

**Comment by bpow** [comment]
I'm not sure exactly what targetInterpretation is supposed to represent... It may have had its name changed from something we would recognize, though. It is hard to tell from version history. Let's discuss on a call and see if @larrybabb can give some insight.

**Comment by larrybabb** [comment]
So this started out with the idea that for each criterion (rule) we would be able to store the default strength, but since the ACMG rules also have multiple directions (interpretation scales), path v benign, I had broken out this direction and called it "targetInterpretation".  (At the time the term direction had not been introduced as far I knew).  

To conflate things, as I got to the more recent point of codifying the "strength" terms I wrestled with decoupling the strength and direction components of the ACMG criterion PM, PVS, BA, BS, etc.. So, I chose not to purify these codes in order to have a clear reference back to the authors of the concepts in the ACMG paper.  In other words, I concluded that while BS = Benign strong and PS = Path strong, which are both STRONG, there is not any real need to have one code.  One might argue that the "Strong" for Path vs Benign are two different levels of Strong which are context sensitive (I don't know for sure).  In addition, Benign uses things like "Stand Alone" = "A" and Path uses "Very Strong" = "VS" and the other interpretation scale or target direction does not.

Not sure if that helps, but that's how this led to this manifestation of the Criterion concept above.

If you'd prefer to decouple the embedded direction in the strength codes, we certainly can.  Additionally, we can change the targetInterpretation to direction or scale.  Scale may be better in the event that we ever come across the need to assess evidence on more than two scales or simply two scales that are not directionally opposed.  We can talk about that last sentence at our meeting.  It is an interesting concept and we probably wouldn't have a given set of rules or criterion that wasn't on a linear scale, so direction may be all that is needed.

**Comment by larrybabb** [comment]
@bpow I realize that we came to a conclusion today regarding the "strength" or "result" codes.  We are using the exact codes defined by ACMG (BA, BS, BP, PP, PM, PS, PVS).  And you suggested we call the field and associated value set "evidence-result" vs the original "evidence-strength".

One thing we did not discuss was the fact that SEPIO separates the notion of direction and strength. I think we want to weigh the desire to stay in sync with SEPIO.  I recall Matt Brush looking at the "direction" concept as something unique to the ClinGen-ACMG model but willing to work with it in an "extensible" way (not as part of core SEPIO).  I do also recall that he considered strength of evidence as a core SEPIO attribute for evidence.  We should verify, but I'd like to re-sync with this concept.

My suggestion is that we work with Matt Brush to verify and possible settle on a compromised name like "weight".  Either that or we just deal with the notion of "strength" being satisfactory, even though it includes the P and B components.

In the end all evidenceLines are assessed (within a single assertion method type) on a linear scale based on the same unit of measure (as defined by the assertion method).  Even though we have called the P and B components of the scale's strength or measure, direction, it is really just a measure of strength or a weight.

So, can we just stick with the attribute name as "strength" and "defaultStrength" for now?
